src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/TestGenieSettingsState.kt

```diff
@@ -43,6 +44,7 @@ data class TestGenieSettingsState(
         val colorBlue: Int = TestGenieDefaultsBundle.defaultValue("colorBlue").toInt()
         val buildPath: String = TestGenieDefaultsBundle.defaultValue("buildPath")
         val buildCommand: String = TestGenieDefaultsBundle.defaultValue("buildCommand")
+        const val telemetryEnabled: Boolean = false
```

Comment:
wouldn't it be better to move this to `TestGenie.Properties`?

Reply:
When I moved `showCoverage` someone suggested that, I remember. Come to think of it, that is the only boolean there

Reply:
(Other team members suggested moving)